CONTRIBUTING.rst

```diff
@@ -223,4 +223,6 @@ Creating a new release
    * Fill in the release description. You can use the "Generate release notes" function to get a draft for this
 3. Commit your changes and push to ``main``
 4. Publish the release
-5. Check that the "publish" `action <https://github.com/litestar-org/litestar/actions>`_ has run successfully
+5. Publish the release.
```

Comment:
this is repeated?

Reply:
Yes?! Just publish it twice, for good measure!

Reply:
Fixed :P